eZ/Publish/Core/Persistence/Legacy/Tests/Content/MapperTest.php

```diff
@@ -251,6 +323,30 @@ public function testExtractContentFromRowsMultipleVersions()
         );
     }
 
+    /**
+     * @param string[] $fields
+     */
+    private function getFieldRegistry(
+        array $fields = [],
+        int $expectedConverterCalls = null
+    ): Registry {
+        $convMock = $this->createMock(Converter::class);
+        $convMock->expects(
+            $expectedConverterCalls === null
+            ? $this->any()
+            : $this->exactly($expectedConverterCalls)
+        )
+            ->method('toFieldValue')
+            ->will($this->returnValue(new FieldValue()));
```

Comment:
This should be sufficient, as it basically does the same thing.
```suggestion
            ->willReturn(new FieldValue());
```